index.js

```diff
@@ -36,6 +36,16 @@ const VM_OPTIONS = {
 };
 const vm = new VM(VM_OPTIONS);
 
+//--- Built-ins
+
+vm.run(
```

Comment:
That is a good point; I have not messed with `vm(2)` regularly enough to think of these things.